child_compassion/mappings/compassion_child_mapping.py

```diff
@@ -16,37 +16,106 @@ class GenericChildMapping(OnrampMapping):
     ODOO_MODEL = 'compassion.generic.child'
 
     CONNECT_MAPPING = {
-        # Fields used in Search Message (for global child)
-        'Beneficiary_GlobalID': 'global_id',
-        'Beneficiary_LocalID': 'local_id',
-        'LocalNumber': 'local_id',
-        'CorrespondentScore': 'correspondent_score',
-        'IsSpecialNeeds': 'is_special_needs',
-        'SourceCode': 'source_code',
-        'PriorityScore': 'priority_score',
-        'FullBodyImageURL': 'image_url',
-        'MinDaysWaiting': 'waiting_days',
 
-        # Fields for DetailedInformation message (for compassion.child)
         'GlobalId': 'global_id',
         'LocalBeneficiaryId': 'local_id',
-
-        # Fields shared for both child types
+        'AcademicPerformance_Name': 'academic_performance',
+        'ActiveProgram': 'type',
         'Age': 'age',
+        'BeneficiaryHousehold': ('household_id', 'compassion.household'),
+        'BeneficiaryHouseholdList': ('household_ids', 'compassion.household'),
+        'BeneficiaryState': 'beneficiary_state',
+        'Beneficiary_Gender': 'gender',
+        'Beneficiary_GlobalID': 'global_id',
+        'Beneficiary_LocalID': 'local_id',
+        'Beneficiary_LocalNumber': 'local_id',
         'BirthDate': 'birthdate',
-        'ICP_ID': ('project_id.icp_id', 'compassion.project'),
+        'Birthdate': 'birthdate',
+        'CDSPType': 'cdsp_type',
+        'ChristianActivity_Name': (
+            'christian_activity_ids.name', 'child.christian.activity'),
+        'ChronicIllness_Name': (
+            'chronic_illness_ids.name', 'child.chronic.illness'),
+        'CognitiveAgeGroup_Name': 'cognitive_age_group',
+        'Community_Name': ('project_id.community_name', 'compassion.project'),
```

Comment:
You can set it to None, because we don't need to set project_id based on community name. Only field ICP_ID is useful for linking the correct project to the child. All other project fields should be ignored. (we will use ICP Get Details message for updating the project values)